testing/scenario_app/lib/src/standard_codec.dart

```diff
@@ -0,0 +1,286 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:math' as math;
+import 'dart:typed_data';
+
+const int __writeBufferStartCapacity = 64;
+
+/// This is mirroring the standard codec from framework
+class StandardMessageCodec {
```

Comment:
I think at some point we need to move standard message codec to some place that is universally accessible and reduce duplication